modules/yams/5.1.3/MODULE.bazel

```diff
@@ -0,0 +1,11 @@
+module(
+    name = "yams",
+    version = "5.1.3",
+    compatibility_level = 1,
+)
+
+bazel_dep(name = "apple_support", version = "1.15.1")
+bazel_dep(name = "rules_swift", version = "1.18.0", repo_name = "build_bazel_rules_swift", max_compatibility_level = 2)
```

Comment:
Hmm, this isn't true though.

@luispadron Since the [repo uses the tag](https://github.com/jpsim/Yams/blob/62b02628d8aff58007cda743598c1f904892fd01/BUILD#L14), it need to be updated to use aspect hints, and bump the minimum supported version.

Reply:
@jpsim

Reply:
`CYaml` exposes a custom modulemap that is found via `includes`, so this continues to work as is.